### PR TITLE
Update filesystem.rst

### PR DESCRIPTION
**Pull request by flip111**
add information about umask to directory creation

### CODE REVIEW

**Comment by flip111** [comment]
Don't know what wrong with the build

```
/home/travis/build/symfony/symfony-docs/components/filesystem.rst:73: ERROR: Unknown target name: "umask<https://en.wikipedia.org/wiki/umask/>".
```

was following http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#embedded-uris-and-aliases

File: components/filesystem.rst [hunk 1]
```diff
@@ -68,6 +68,10 @@ On POSIX filesystems, directories are created with a default mode value
 
     This function ignores already existing directories.
 
+.. note::
+
+    The directory permissions are affected by the current `umask<https://en.wikipedia.org/wiki/Umask/>`_.. Set the umask for your webserver, use PHP's `umask() function<https://secure.php.net/umask>`_. or use the chmod() function after the directory has been created.
```
Comment (xabbuh) [inline components/filesystem.rst]: There are few things to change here, but overall this looks good. Can you update the paragraph to something like this:

```rst
.. note::

    The directory permissions are affected by the current `umask`_. Set
    the umask for your webserver, use PHP's :phpfunction:`umask` function
    or use the :phpfunction:`chmod` function after the directory has been
    created.
```

You then need to add the link target at the end of the document:

```rst
.. _`umask`: https://en.wikipedia.org/wiki/Umask
```

**Review by javiereguiluz — APPROVED** [review]

File: components/filesystem.rst [hunk 2]
```diff
@@ -68,6 +68,10 @@ On POSIX filesystems, directories are created with a default mode value
 
     This function ignores already existing directories.
 
+.. note::
+
+    The directory permissions are affected by the current `umask`_. Set the umask for your webserver, use PHP's :phpfunction:`umask` function or use the :phpfunction:`chmod` function after the directory has been created.
```
Comment (xabbuh) [inline components/filesystem.rst]: Please also wrap all lines after the first word that crosses the 72nd character.

**Comment by xabbuh** [comment]
Thank you @flip111.

**Pull request closed without merging**